fixtures/Gemfile-github-pages

```diff
@@ -1,3 +1,5 @@
 source "https://rubygems.org"
 
 gem 'github-pages', group: :jekyll_plugins
+
+gem "jekyll-include-cache", group: :jekyll_plugins
```

Comment:
Turns out, this isn't necessary! `jekyll-include-cache` is actually [already part of `github-pages`](https://pages.github.com/versions/).

```suggestion
```

Reply:
Committing the suggested change seems to have made the github-pages build fail.

Reply:
Interesting, the [failing CI log indicates that `jekyll-include-cache` is being installed](https://github.com/just-the-docs/just-the-docs/actions/runs/5673497486/job/15375074566?pr=1244#step:4:131). Locally, I was able to replace this build when I made the original comment, but I now can't; I get the same failure with

```
$ BUNDLE_GEMFILE=fixtures/Gemfile-github-pages bundle
$ BUNDLE_GEMFILE=fixtures/Gemfile-github-pages bundle exec jekyll build
```

I'm a bit confused *why* this happens. I can try with a brand-new site that uses your branch as a remote theme, and see if it builds on GitHub's stock (i.e. non-explicit Actions) pages runner - that would be the ultimate test.